build/configs/artik053/README.md

```diff
@@ -116,11 +119,9 @@ SUBSYSTEMS=="usb",ATTRS{idVendor}=="0403",ATTRS{idProduct}=="6010",MODE="0666" R
 
 ## How to program a binary
 
-There are two methods, using OpenOCD or script.
-
-After building TizenRT, follow below steps at $TIZENRT_BASEDIR/os folder.
-
-TIZENRT_BASEDIR was set at [[Getting the sources]](../../../README.md#getting-the-sources) tab of Quick Start.
+There are two methods, using OpenOCD or script.  
```

Comment:
would be nice to distinguish between the two methods. As in, to tell when one method would be preferable over another.